rbc/tests/test_omnisci_column_basic.py

```diff
@@ -550,6 +550,10 @@ def test_column_aggregate(omnisci, prop, oper):
     omnisci.reset()
     omnisci.register()
 
+
+    if prop == 'groupby' and oper == 'corr':
+        pytest.skip('omniscidb server crashes')
```

Comment:
This is likely omniscidb version dependent. Could you create an issue about the crash and related details what are available?

Reply:
I've seen this failure in other builds. It seems to be a random thing.